src/collectors/httpjson/httpjson.py

```diff
@@ -27,7 +28,8 @@ def get_default_config(self):
         default_config = super(HTTPJSONCollector, self).get_default_config()
         default_config.update({
             'path': 'httpjson',
-            'url': 'http://localhost/stat'
+            'url': 'http://localhost/stat',
+            'headers': {'User-Agent': 'Diamond HTTP collector'},
```

Comment:
I'm pretty happy with this change in general / prepping to merge this, but I'd like to second this question. Can you provide an example configuration snippet to confirm what it looks like?